packages/@sanity/portable-text-editor/test/__tests__/writingTogether.collaborative.test.ts

```diff
@@ -0,0 +1,421 @@
+/**
+ * @jest-environment ./test/setup/collaborative.jest.env.ts
+ */
+
```

Comment:
This is how we make a collaborative test by setting `@jest-environment ./test/setup/collaborative.jest.env.ts`

This exposes global test functions like `setDocumentValue` `getEditors` to be used inside each test:
```
type Editor = {
  editorId: string
  focus: () => Promise<void>
  getSelection: () => Promise<EditorSelection | null>
  getValue: () => Promise<Value>
  insertNewLine: () => Promise<void>
  insertText: (text: string) => Promise<void>
  pressKey: (keyName: string, times?: number) => Promise<void>
  setSelection: (selection: EditorSelection | null) => Promise<void>
  testId: string
}
declare global {
  function getEditors(): Promise<Editor[]>
  function setDocumentValue(value: Value): void
}
```